src/lib/DataTableHeader/DataTableRow.tsx

```diff
@@ -22,14 +23,54 @@ export function DataTableRow<T, TRouteNames>({
   columns,
   actions,
   rowStyle,
+  rowHighlight,
   actionsPosition,
 }: DataTableRowProps<T, TRouteNames>) {
   const keyValue = getDeepValue(record, keyField);
   const [collapsed, setCollapsed] = useState(true);
 
+  var operator_table = {
+    ">": function (a: number | Date, b: number | Date) {
+      return a > b;
+    },
+    "<": function (a: number | Date, b: number | Date) {
+      return a < b;
+    },
+    "==": function (a: number | Date, b: number | Date) {
+      return a == b;
+    },
+    "!=": function (a: number | Date, b: number | Date) {
+      return a != b;
+    },
+  };
+
+  function setStyle(rowObjectT: T, rowHighlight?: RowHighlightInterface<T>): CSSProperties | undefined {
```

Comment:
`getStyle` would be a better name for this.

Reply:
Fixed!